changelog/7760.breaking.rst

```diff
@@ -0,0 +1,2 @@
+`sunpy.map.GenericMap.contour` now internally uses ContourPy instead of scikit-image for improved performance.
```

Comment:
Can the text in this changelog be adjusted for Cadair's comment above?

Reply:
Is it fine now?